source/p_spec.c

```diff
@@ -2172,6 +2172,7 @@ void P_PlayerInSpecialSector (player_t* player)
 
       case 11:
         if (!(_g->leveltime&0x1f))
+          _g->player.cheats -= CF_GODMODE; //Vanilla disables GodMode in floor-11
```

Comment:
I think this if-statement is missing curly braces.

It looks a bit different in the original Linux Doom source:
https://github.com/id-Software/DOOM/blob/77735c3ff0772609e9c8d29e3ce2ab42ff54d20b/linuxdoom-1.10/p_spec.c#L1054